src/client/Client.h

```diff
@@ -601,6 +601,9 @@ class Client : public Dispatcher, public md_config_obs_t {
   void set_filer_flags(int flags);
   void clear_filer_flags(int flags);
 
+  void set_uuid(const std::string& uuid, bool reclaim=false);
```

Comment:
The missing piece here is that this interface does not tell the client whether reclaim succeeded. Ganesha needs to know that in order to know how to proceed with reclaim. The set_reclaim_done function should be fine though.

Reply:
set reclaim uuid before mount. If mount succeeds, then reclaim succeeds. does this make sense?

Reply:
Hmm ok, that might work. So on ganesha startup we'd do:
```
set_uuid(uuid, true);
ceph_mount(...);
```
...then if that fails with the error that indicates that reclaim didn't work, we do:
```
set_uuid(uuid, false);
ceph_mount(...);
```
...it's a little kludgey but that should work.